time stamp
barrett_hw/arm_cartesian_state[] robot_cartesian_state
